src/nft/components/explore/Explore.css.ts

```diff
@@ -0,0 +1,120 @@
+import { style } from '@vanilla-extract/css'
+import { breakpoints, sprinkles } from 'nft/css/sprinkles.css'
+
+/* Override for BodyWrapper properties */
+export const fullWidth = style([
+  sprinkles({
+    left: '1/2',
+    right: '1/2',
+    width: 'full',
+    maxWidth: 'full',
+    position: 'absolute',
+  }),
+  {
+    top: '72px',
+    marginLeft: '-50vw',
+    marginRight: '-50vw',
+  },
+])
```

Comment:
does 100vw positioned at left:0 not work here? This looks at first glance like it's a convoluted center/fill.